### PR TITLE
Build the search functionality

### PR DESCRIPTION
**Pull request by petercrackthecode**
Implemented the search feature. Will need to style the dataset cards.

### CODE REVIEW

File: src/views/Buy/Buy.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  CardActions,
+  CardContent,
+  Grid,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import Header from "components/Header/Header";
+import React, { useEffect, useState } from "react";
+import { database } from "../../firebase/index";
+import "./Buy.css";
```
Comment (Abhishek-kumar09) [inline src/views/Buy/Buy.js:13]: Change it to `./Buy.scss`
`css.map` is not working, infact it doesn't worked on my machine too.

**Comment by Abhishek-kumar09** [comment]
@petercrackthecode Cant find the file, 
image

**Comment by Abhishek-kumar09** [comment]
@petercrackthecode I will fix it.

**Comment by petercrackthecode** [comment]
Thank you so much @Abhishek-kumar09 !

**Comment by Abhishek-kumar09** [comment]
See the deployed version, something is broken auction page not working. and there is change ins some styles on homepage too for 100% zoom in my browser.

**Comment by Abhishek-kumar09** [comment]
@petercrackthecode This is how it looks on my browser
image

**Pull request closed without merging**